ZT - Cloudflare Dashboard session management SSO (#7736)
cloudflare/cloudflare-docs: content/fundamentals/account-and-billing/account-security/manage-active-sessions.md

{{<Aside type="note">}}

By default, the session timeout for the [Cloudflare dashboard](https://dash.cloudflare.com) and [Cloudflare Zero Trust dashboard](https://dash.teams.cloudflare.com/) is 24 hours. 

> By adding a Dashboard SSO application to your Cloudflare Zero Trust account, you can enforce single sign-on (SSO). Once you have configured SSO

Reviewer: ```suggestion

Some customers can also enforce single-sign on (SSO) by [adding a Dashboard SSO application](/cloudflare-one/applications/configure-apps/dash-sso-apps/).
```

> escalate to your Customer Success Manager for Cloudflare Dashboard SSO and the timeout to be configured.

Reviewer: ```suggestion
```
